lightkurve/periodogram.py

```diff
@@ -2,64 +2,251 @@
 from astropy import units as u
```

Comment:
Let's add one or two lines of comments at the top of `periodogram.py` just to explain what the module is for.

Reply:
You mean with regards to astropy.units? I have included:

"""This module lets us attack a unit to a value or an array of values. This
allows us to keep track of what units our data are in, and easily switch
between different units."""
from astropy import units as u